tests/plugins/tableselection/manual/integrations/image/nestedimage.md

```diff
@@ -0,0 +1,27 @@
+@bender-ui: collapsed
+@bender-tags: bug, 2235, 4.11.5
+@bender-ckeditor-plugins: wysiwygarea, tableselection, image, undo
+
+## Context menu for image in table
+
+1. Open context menu for image by right-clicking it.
+2. Choose `Image properties`.
+
+**Expected result:**
+
+URL field should be filled and image should be visible in the preview.
+
+**Unexpected result:**
+
+URL field is empty or image isn't visible in the preview.
+
+3. Select first row of the outer table and delete it.
```

Comment:
This steps are hardly visible in the scenario. Maybe you could just create two separate scenarios here, e.g:

```md
## Context menu for image in table

### Scenario 1

1. ...

Expected...
Unexpected...

### Scenario 2

1. ...

Expected...
Unexpected...

```